WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingTaskView.swift

```diff
@@ -70,11 +84,19 @@ struct StoreOnboardingTaskView: View {
 }
 
 private extension StoreOnboardingTaskView {
+    enum Localization {
+        static let privateLabel = NSLocalizedString("PRIVATE", comment: "Label shown on the launch store task.")
```

Comment:
Nit: we cannot trust translators to keep the text all caps - how about using lowercase and applying `uppercased()` to the string in code?

Reply:
TIL. Done in 90090e80278df99ac007c7381461a8efe12ea80f